### PR TITLE
[EXPERIMENT][WIP] Streamline Appliance, DB & SSH loggers

### PR DESCRIPTION
**Pull request by mfalesni**

### CODE REVIEW

File: utils/appliance/__init__.py [hunk 1]
```diff
@@ -142,6 +148,36 @@ def uninstall_ipa_client(self):
...
+class RESTLoggerAdapter(logging.LoggerAdapter):
+    def process(self, msg, kwargs):
+        return '[REST]: {}'.format(msg), kwargs
+
+
+class SSHLoggerAdapter(logging.LoggerAdapter):
+    def process(self, msg, kwargs):
+        return '[SSH]: {}'.format(msg), kwargs
+
+    def trace(self, msg, *args, **kwargs):
```
Comment (RonnyPfannschmidt) [inline utils/appliance/__init__.py]: can we avoid trace support instead, perhaps even use a different logger with different level

File: utils/appliance/__init__.py [hunk 2]
```diff
@@ -142,6 +148,36 @@ def uninstall_ipa_client(self):
         assert return_code != 0
 
 
+class ApplianceLoggerAdapter(logging.LoggerAdapter):
```
Comment (RonnyPfannschmidt) [inline utils/appliance/__init__.py]: can we generalize tagged log adaption?
where tag == adding `[...]` in front of messages, adding a spacer if necessary

File: utils/appliance/__init__.py (hunk 1 above)
Comment (mfalesni) [inline utils/appliance/__init__.py]: Sure, I added this just due to the ssh client using it.

File: utils/appliance/__init__.py (hunk 2 above)
Comment (mfalesni) [inline utils/appliance/__init__.py]: Yes, I was thinking about it. Some sort of StackingTagAddingAdaptor with a class varialbe specifying the tag added.

File: utils/appliance/__init__.py (hunk 1 above)
Comment (RonnyPfannschmidt) [inline utils/appliance/__init__.py]: i guess thats acceptable, we should take a look at moving ssh logging to a own logger

File: utils/appliance/__init__.py (hunk 2 above)
Comment (mfalesni) [inline utils/appliance/__init__.py]: I wonder - can we extend the data passed through the adaptors? Eg we would have `tags`, appliance would add `appliance`, ssh client would add `ssh` et cetera - so once we start processing the logs in some sort of elk/splunk, we can have all the tags from all the adaptors the message passed through? @RonnyPfannschmidt

Comment (RonnyPfannschmidt) [inline utils/appliance/__init__.py]: we should take a look at what happens with the kwargs, and extend on that

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: utils/appliance/implementations/ui.py [hunk 3]
```diff
@@ -353,19 +365,15 @@ def pre_navigate(self, *args, **kwargs):
...
-        str_msg = "[UI-NAV/{}/{}]: {}".format(self.obj.__class__.__name__, self._name, msg)
-        getattr(logger, level)(str_msg)
-
-    def construst_message(self, here, resetter, view, duration):
+    def construct_message(self, here, resetter, view, duration):
         str_here = "Already Here" if here else "Needed Navigation"
         str_resetter = "Resetter Used" if resetter else "No Resetter"
         str_view = "View Returned" if view else "No View Available"
-        return "{}/{}/{} (elapsed {}ms)".format(str_here, str_resetter, str_view, duration)
+        return ("%s/%s/%s (elapsed %d ms)", str_here, str_resetter, str_view, duration)
```
Comment (RonnyPfannschmidt) [inline utils/appliance/implementations/ui.py]: why that exchange?

Comment (RonnyPfannschmidt) [inline utils/appliance/implementations/ui.py]: forget that, it took a second to notice the return value tuple

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by mfalesni** [comment]
@RonnyPfannschmidt @psav Do you think this has any value moving forward or shall I close this PR?

**Pull request closed without merging**